src/api/comments.ts

```diff
@@ -0,0 +1,37 @@
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getComments(postId: number) {
+  const response = await fetch(`${BASE_URL}/comments?postId=${postId}`);
```

Comment:
`${BASE_URL}/comments?postId=${postId}` you can move it to a const variable `apiEndpoint`

Improve this in all other asynchronous functions